doc/api/esm.md

```diff
@@ -83,12 +83,20 @@ All CommonJS, JSON, and C++ modules can be used with `import`.
 Modules loaded this way will only be loaded once, even if their query
 or fragment string differs between `import` statements.
 
-When loaded via `import` these modules will provide a single `default` export
-representing the value of `module.exports` at the time they finished evaluating.
+CommonJS modules, when imported, will be handled in one of two ways. By default
+they will provide a single `default` export representing the value of
+`module.exports` at the time they finish evaluating. However, they may also
+provide `@@esModule` or `__esModule` to use named exports, representing each
```

Comment:
What’s the motivation for keeping `__esModule` when a symbol is available? You can always `import { esModule } from 'module';`, right?

Reply:
Or, just a thought: What if this was an alternative to `module.exports`, e.g. `module.esmExports`? We could check for the presence of that property when the script finished.

The upsides of this would be that you don’t need a symbol, and you don’t need to modify the actual exports object